samples/bot-consent-azuread/csharp/Microsoft.Teams.Samples.Bot.Consent.AzureAD/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "MicrosoftAppId": "",
+  "MicrosoftAppPassword": "",
+  "BaseUrl": "https://ENTERDOMAINHERE.ngrok.io/",
+  "BotAuthConnectionName": "AAD"
```

Comment:
I feel we should update ReadMe to guide developers to create connection string with name "AAD".  Thoughts?